### PR TITLE
Add JwtCreator for signing and encrypting JWT

### PR DESCRIPTION
**Pull request by iriale**

### CODE REVIEW

File: shared/src/main/java/org/pac4j/lagom/jwt/JwtHeaderHandlerHelper.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        if (signatureConfiguration == null) {
+            jwt = new PlainJWT(claimsSet);
+        } else {
+            jwt = signatureConfiguration.sign(claimsSet);
+        }
+        return encryptionConfiguration != null ? encryptionConfiguration.encrypt(jwt) : jwt.serialize();
+    }
+
+    /**
+     * Генерация заголовка с хедером авторизации.
```
Comment (stankevichevg) [inline shared/src/main/java/org/pac4j/lagom/jwt/JwtHeaderHandlerHelper.java:73]: I think not all here speak Russian )

Comment (iriale) [reply shared/src/main/java/org/pac4j/lagom/jwt/JwtHeaderHandlerHelper.java]: of course you're right )

**Comment by ihostage** [comment]
Thank you, @iriale!
I will review this PR as soon as I can!

**Comment by leleuj** [comment]
@iriale Can you sync with master? Thx

**Comment by ihostage** [comment]
@leleuj Sorry about that... This could have been a part of `2.2.0`, but I rushed 😞  I think, that after some changes we will need to cut the next release.

**Comment by leleuj** [comment]
No problem, let's sync and review. I'll merge after that and release a v2.2.1.

**Review by ihostage — APPROVED** [review]
@leleuj I think, that this PR ready to review and merge 😄 
But don't cut `2.2.1`, please. I need time to test a snapshot version in our project. 😉

**Comment by leleuj** [comment]
@ihostage I have one question: why not re-use the `JwtGenerator` already available in pac4j?

**Comment by ihostage** [comment]
Very good question 👍  Thank a lot, @leleuj! 
I updated PR and re-used the `JwtGenerator`.

**Comment by leleuj** [comment]
I forgot to mention that we also have a `JWKHelper`, you may want to use it as well. Sorry for the partial information.

**Comment by ihostage** [comment]
Thank you, @leleuj! 
I analyzed this class and think, that I will keep `com.nimbusds.jose.jwk.JWK#parse` as universal parser JWK. Because I don't know `kty` before JWK will be parsed from config. This method is more useful for our case.

**Pull request merged**